client/src/plus/ai-help/login-banner.tsx

```diff
@@ -14,7 +14,7 @@ export function AiLoginBanner() {
       <span>
         <span>Want to use AI Help?</span>{" "}
         <strong>
-          Upgrade to{" "}
+          Log in to{" "}
```

Comment:
UX comment: same as above, weird duplication of "log in", and I believe this sets the wrong expectation of being able to log in with the link following:

![Screen Shot 2023-07-21 at 17 28 07](https://github.com/mdn/yari/assets/755354/f2eacc30-d040-4c59-bd8f-2b740ea39548)

Reply:
Changed to "Upgrade to MDN Plus for free", like it was in the Playground.